src/app/static/src/app/components/ResetConfirmation.vue

```diff
@@ -6,7 +6,7 @@
 
             <p v-translate="'discardWarning'"></p>
             <ul>
-            <li v-for="step in changesToRelevantSteps" :key="step.number">
```

Comment:
Please remove this file from this PR - the change doesn't seem relevant to the ticket in question (see [PR guidelines](https://mrc-ide.myjetbrains.com/youtrack/articles/RESIDE-A-13/Pull-Request-guidelines))